.github/actions/deploy-beta/action.yml

```diff
@@ -105,4 +111,13 @@ runs:
       env:
         TWILIO_ACCOUNT_SID: ${{ inputs.account-sid }}
         TWILIO_AUTH_TOKEN: ${{ inputs.auth-token }}
-      shell: bash
\ No newline at end of file
+      shell: bash
+    # Send Slack notifying success
+    - name: Slack Aselo channel
+      id: slack
+      uses: slackapi/slack-github-action@v1.14.0
+      with:
+        channel-id: "C02L11XCKQU"
```

Comment:
This doesn't seem all that sensitive, but would it be better as a GH secret?

Reply:
This is something I have a question about: this channel id is for the channel I've created. Should we use one of the existing ones or create a new one? Then I will move this to be a secret if preferred.

Reply:
I think having an #aselo-deploys channel like you created is a great idea. Then it doesn't clutter other channels.